subprojects/docs/src/samples/signing/conditional/conditionalSigning.sample.conf

```diff
@@ -2,8 +2,12 @@ commands: [{
     execution-subdirectory: groovy
     executable: gradle
     args: tasks
+    # Do not fail for deprecation warnings: deprecated Maven plugin
+    flags: "--warning-mode=none"
 }, {
     execution-subdirectory: kotlin
     executable: gradle
     args: tasks
+    # Do not fail for deprecation warnings: deprecated Maven plugin
+    flags: "--warning-mode=none"
```

Comment:
Why are you using `--warning-mode=none` in this sample and `--warning-mode=all` in the `useMavenPlugin.sample.conf`?

Reply:
I have been using `none` for the ones that check their output, and tried to use `all` for the rest, but wasn't always very consistent.